blocks/input/input.css

```diff
@@ -0,0 +1,16 @@
+.input{
```

Comment:
@wwily Since the input class doesn't have the `align-items` property assigned, as a flex container, it tries to stretch all child elements to 100%, including the label.

The issue arises when I hover very right on the label's space, where there is no text, and our input changes border color, which isn't ideal from a UI/UX perspective and can confuse our users (see the video).


https://github.com/p-foundation/pink-app/assets/76534205/eb10e3d1-042c-4bca-b847-6fa6a3dc1080



Let's fix this by assigning `align-items: flex-start;` to the input.

Reply:
done 😌